src/syslogwrappers.cpp

```diff
@@ -85,9 +85,9 @@ dmtcp_Syslog_PluginDescr()
 static string&
 _ident()
 {
-  static string t;
+  static string *t = new string();
```

Comment:
Doesn't this create a memory leak by allocating in the heap, instead of tmpdir on the stack?